_sass/charts.scss

```diff
@@ -3,11 +3,11 @@ rect.bar {
   mix-blend-mode: difference;
 }
 
-text {
-  fill: rgb(71, 71, 71);;
-  mix-blend-mode: luminosity;
-  font-family: 'Source Code Pro', monospace;
-}
+// text {
+//   fill: rgb(71, 71, 71);;
+//   mix-blend-mode: luminosity;
+//   font-family: 'Source Code Pro', monospace;
```

Comment:
can we keep the font-family attribute on the text?

Reply:
No cause otherwise this graph is not very readable 
![image](https://user-images.githubusercontent.com/12419401/123460864-4a2a9180-d5b6-11eb-8185-b15de0ba836f.png)
We don't know which line is which label

Reply:
just the font-family -- not the fill colour. i played around with charts.scss and got to get this result: 
![image](https://user-images.githubusercontent.com/11150920/123465932-bb6d4300-d5bc-11eb-89e1-715f6aab5a6b.png)